.gitpod/scripts/login.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if [ -z "$PLATFORM_CLI_TOKEN" ]; then
+    echo "\$PLATFORMSH_CLI_TOKEN was detected as an environment variable. The Platform CLI will authenticate using that token.";
+    echo "You may updated it with: eval $(gp env -e PLATFORMSH_CLI_TOKEN=NEW_TOKEN)";
```

Comment:
```suggestion
    echo '$PLATFORMSH_CLI_TOKEN was detected as an environment variable. The Platform CLI will authenticate using that token.'
    echo 'You may update it with: eval $(gp env -e PLATFORMSH_CLI_TOKEN=NEW_TOKEN)'
```

But, I'm not sure we'd want to recommend this. Gitpod environments can be shared, and the token would then be shared. As far as I can tell, people can authenticate without this script, by just setting the variable, if they know what they are doing.